proposals/context-receivers.md

```diff
@@ -278,11 +278,12 @@ signature of the functional type replicates the textual order in which every arg
   }
   ```
 
-### Referencing specific context receiver
+### Referencing specific receiver
```

Comment:
Table of contents needs to be fixed, too. I'm using knit to generate it -> Clone https://github.com/Kotlin/kotlinx-knit, open it in IDEA, num Knit main file, pass it it the path to the file that needs processing.

Reply:
Thanks, updated